rosenpass/src/cli.rs

```diff
@@ -215,21 +224,27 @@ pub enum CliCommand {
         force: bool,
     },
 
-    /// Deprecated - use gen-keys instead
+    /// Validate a configuration file
+    ///
+    /// This command will validate the configuration file and print any errors
+    /// it finds. If the configuration file is valid, it will print a success.
+    /// Defined secret & public keys are checked for existence and validity.
+    Validate { config_files: Vec<PathBuf> },
+
+    /// DEPRECATED - use the gen-keys command instead
     #[allow(rustdoc::broken_intra_doc_links)]
     #[allow(rustdoc::invalid_html_tags)]
+    #[command(hide = true)]
     Keygen {
-        // NOTE yes, the legacy keygen argument initially really accepted "privet-key", not "secret-key"!
+        // NOTE yes, the legacy keygen argument initially really accepted
+        // "privet-key", not "secret-key"!
```

Comment:
```suggestion
        // NOTE yes, the legacy keygen argument initially really accepted
        // "private-key", not "secret-key"!
```

I assume this is just a long-standing typo in the comment, and that the CLI actually needed `private-key`? 😅

Reply:
Fixed.